src/GraphBLAS-sharp/Backend/Common/Utils.fs

```diff
@@ -1,7 +1,7 @@
 namespace GraphBLAS.FSharp.Backend.Common
 
 module internal Utils =
-    let workGroupSize = 128
+    let workGroupSize = 256
```

Comment:
What the magic constant? Can we load real workgroup size from the accelerator?

Reply:
We can take the maximum size of the working group from the device, but the optimal size for a particular algorithm is determined empirically. Usually, it is 256

Reply:
Well. But looks strange, because it should depend on the real size of local memory and the number of threads.